test/test_rb.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A Tester for the RB experiment
+"""
+
+from qiskit.quantum_info.operators.predicates import matrix_equal
+from qiskit.quantum_info import Clifford
+from qiskit.quantum_info.operators.measures import process_fidelity
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeParis
+from qiskit.exceptions import QiskitError
+import numpy as np
+import qiskit_experiments as qe
+
+
+class TestRB(QiskitTestCase):
+    """
+    A simple and primitive backend, to be run by the RB tests
+    """
+
+    def rb_parameters_one_qubit(self):
+        """
+        Initialize data and executing RB experiment on one qubits with specific parameters
+        Returns:
+            dict: A dictionary with the experiment setup attributes.
+            RBExperiment: The instance for the experiment object.
+            ExperimentData: The experiment data and results after it had run.
+        """
+        backend = FakeParis()
+        exp_attributes = {
+            "qubits": [1],
+            "lengths": [1, 3, 5, 7, 9],
+            "num_samples": 1,
+            "seed": 100,
+        }
+        rb = qe.randomized_benchmarking
+        rb_exp = rb.RBExperiment(
+            exp_attributes["qubits"],
+            exp_attributes["lengths"],
+            num_samples=exp_attributes["num_samples"],
+            seed=exp_attributes["seed"],
+        )
+        exp_data = rb_exp.run(backend)
+        exp_circuit = rb_exp.circuits()
```

Comment:
These is not a single circuit but a list of circuits, so use `exp_circuits =`

Reply:
Changed it to "circuits"